server/functions.lua

```diff
@@ -90,10 +90,18 @@ end
 --- @param itemName string The name of the item to search for.
 --- @param quality number item quality to match
 --- @return number|nil - The slot number of the first matching item, or nil if no match is found.
-Inventory.GetFirstSlotByItemWithQuality = function(items, itemName, quality)
+Inventory.GetFirstSlotByItemWithQuality = function(items, itemName, quality, serial)
```

Comment:
not sure where this function came from, should be `item quality >= parameter`, not equals. but adding serial makes even less sense